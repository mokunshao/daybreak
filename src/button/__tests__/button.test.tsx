import { render } from '@testing-library/react';
import React from 'react';
import Button from '../button';

describe('button', () => {
  it('是个 button', () => {
    const { asFragment } = render(<Button />);
    expect(asFragment()).toMatchSnapshot();
  });
});
