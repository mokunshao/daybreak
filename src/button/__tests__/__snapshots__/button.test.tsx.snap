// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是个 button 1`] = `
<DocumentFragment>
  <button
    class="daybreak-button daybreak-button-normal"
    type="button"
  />
</DocumentFragment>
`;
